src/controllers/CreateFormController.js

```diff
@@ -18,13 +18,13 @@ module.exports = class CreateFormController {
   }
 
   post = async (req, res) => {
-    const form = this.getForm(req.body);
+    const form = this.getForm(req.body.form);
 
     if (FormValidator.isValid(form)) {
       const id = await this.formRepo.save(form);
       await this.resultsRepo.save(this.createEmptyResults(form, id));
       console.log(`Form created:`, form);
-      res.send(id);
+      res.send(id.toString());
```

Comment:
Jak to był Number to express jakieś dziwne errory rzucał (których nie rzucał przy testach, ale to JS, więc jakoś mnie to nie dziwi XD). Z drugiej strony właściwe id z mogo będzie stringiem, więc nawet lepiej, żeby to stringa zwracało.